src/qt/intro.cpp

```diff
@@ -322,6 +305,9 @@ void Intro::startThread()
 
     connect(executor, &FreespaceChecker::reply, this, &Intro::setStatus);
     connect(this, &Intro::requestCheck, executor, &FreespaceChecker::check);
+    connect(ui->prune, &QCheckBox::stateChanged, [this](int state) {
+        setSizeWarningLabel(state == 2);
+    });
```

Comment:
This does not work if "-prune" is set:

```bash
src/qt/bitcoin-qt -prune=3000
```

![Screenshot from 2019-10-05 11-15-01](https://user-images.githubusercontent.com/32963518/66252279-4b70b680-e762-11e9-80a5-fb9568535f5f.png)

Reply:
255ab91b31ca3848654a63fcc384b83913bd36e8